pagopa/pn-frontend-e2e
# Pn 9165 e 2 e destinatario pg delegante p giuridica amministratore aggiunta nuova delega

## Short description

PN - 9165 refactor test PG aggiunta nuova delega

## List of changes proposed in this pull request

created new step to wait the expiration of OTP code for acceptance of the delegation
implemented control on delegation creation form

## How to test

On the delegation page, click on button "aggiungi una delega", verify every form field to be correct, create a new delegation, then wait for the acceptance code to expire and verify the delegation table is empty.

- [ x ] Does this PR require a manual test in AWS CodeBuild?
